bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TeaserImpl.java

```diff
@@ -333,7 +341,8 @@ private Action(Resource actionRes, String parentId) {
             ctaTitle = ctaProperties.get(PN_ACTION_TEXT, String.class);
             ctaUrl = ctaProperties.get(PN_ACTION_LINK, String.class);
             ctaPath = actionRes.getPath();
-            if (ctaUrl != null && ctaUrl.startsWith("/")) {
+            if (ctaUrl != null) {
```

Comment:
Why remove the path restriction? I don't know of a situation in which the page manager will return a non-null value for a path that starts with anything other than `/`.